Extensions/mass_deleter.js

```diff
@@ -293,9 +293,11 @@ XKit.extensions.mass_deleter = new Object({
 
 	unlike_next_max: 30,
 	unlike_next_current: 0,
+	unlike_next_page_url: "",
```

Comment:
should probably be `undefined` if we're expecting it not to be used anywhere. Otherwise should be `/likes` or something?